frappe/website/web_template/section_with_right_image/section_with_right_image.html

```diff
@@ -0,0 +1,28 @@
+<div class="section-with-right-image">
```

Comment:
This template is very similar to Split Section with Image template. The difference is the content / image split is 6 / 6 in current one and this one has 8 / 4.

I think a better way to handle is to add a select option to let user choose between the splits.
The options could be:
- 6 / 6
- 7 / 5
- 8 / 4
- 9 / 3

Reply:
Also there is no subtitle field, is it fine if I add subtitle field as well?